strictdoc/backend/sdoc_source_code/reader_robot.py

```diff
@@ -0,0 +1,198 @@
+# mypy: disable-error-code="no-redef,no-untyped-call,no-untyped-def,type-arg,var-annotated"
```

Comment:
Let's make sure we add this new code with all mypy issues resolved. Many files still have these mypy ignores but we have been working towards cleaning them all up.

Reply:
Thanks, full ack.